DMX.Core.Api.Tests.Unit/Services/Foundations/LabCommands/LabCommandServiceTests.Logic.Modify.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using DMX.Core.Api.Models.Foundations.LabCommands;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Xunit;
+
+namespace DMX.Core.Api.Tests.Unit.Services.Foundations.LabCommands
+{
+    public partial class LabCommandServiceTests
+    {
+        [Fact]
+        public async Task ShouldModifyLabCommandAsync()
+        {
+            // given
+            LabCommand randomLabCommand = CreateRandomLabCommand();
+            LabCommand inputLabCommand = randomLabCommand;
+            LabCommand storageLabCommand = inputLabCommand;
+
+            DateTimeOffset randomDate = GetValidDateTimeOffset(
+                inputLabCommand.CreatedDate,
+                new TimeSpan(0, 1, 0));
+
+            DateTimeOffset currentDate = randomDate;
+            inputLabCommand.UpdatedDate = currentDate;
```

Comment:
I don't know if all of this is necessary - just generate a random date in the space, push your created date backgrounds and move on.